vmdb/app/assets/javascripts/miq_dhtmlxgrid.js

```diff
@@ -200,11 +200,11 @@ function miqInitGrid(grid_name) {
 function miqOnAECheck(row_id, cell_idx, state) {
   crows = this.getCheckedRows(0);
   $("#miq_grid_checks").each(function () {
-    this.value = crows
-  })
+    this.value = crows;
+  });
```

Comment:
Given `$('#foo')` should never return more than one element, maybe `$('#miq_grid_checks').val(crows);` ?